plugins/accuracy.h

```diff
@@ -28,7 +28,7 @@ struct AccuracyConfig : public ProcedureConfig {
     static constexpr const char * name = "classifier.test";
 
     AccuracyConfig()
-          : mode(CM_BOOLEAN), uniqueScoresOnly(false)
+          : mode(CM_BOOLEAN), uniqueScoresOnly(false), accuracyOverN(1)
```

Comment:
They were written before C++11.  Now we're moving to in-member wherever possible; we should fix where we can.